sample-input/benchmarks/c5g7/c5g7-3d-rodded-B.py

```diff
@@ -0,0 +1,526 @@
+import openmoc
+
+###############################################################################
```

Comment:
Can this be turned into a "geometry3d.py" file which can be imported into "c5g7.py", "c5g7-gpu.py" or "c5g7-cmfd.py"? Just doesn't seem to align with the format we have switched to which minimized code reuse by abstracting the geometry construction into a single shared and importable Python module.

Or perhaps you could take this geometry construction code and append it to "geometry.py" and name the two different `Geometry` objects as `geometry2d` and `geometry3d`, either of which could be imported from `geometry` and added to a `TrackGenerator`.